src/System.Reflection.Metadata/tests/TestUtilities/LoaderUtilities.cs

```diff
@@ -0,0 +1,36 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Microsoft.Win32.SafeHandles;
+using System.IO;
+using System.Reflection.PortableExecutable;
+using Xunit;
+
+namespace System.Reflection.Metadata.Tests
+{
+    internal unsafe static class LoaderUtilities
+    {
+        public static void LoadPEAndValidate(byte[] peImage, Action<PEReader> validator)
```

Comment:
@stephentoub @jkotas How do I do this on Linux/Mac? That is, load a PE file and get its address for testing purposes? I assume the loader logic is implemented in CoreCLR, unlike on Windows where it's the OS loader that's laying out the PE sections. Correct?